src/Packs.sol

```diff
@@ -364,7 +363,13 @@ contract Packs is
         // Handle user choice and fulfil order or payout
         if (fulfillmentType == FulfillmentOption.NFT) {
             // execute the market data to transfer the nft
-            bool success = _fulfillOrder(marketplace_, orderData_, orderAmount_);
+            bool success = false;
+            try this._fulfillOrder(marketplace_, orderData_, orderAmount_) returns (bool result) {
```

Comment:
This is more verbose but an unnecessary try/catch. This actually happens occasionally on Lucky Buy https://etherscan.io/tx/0x65049622ca3a58c529ed4fb263d3f0ebd8ee5a428b464c3b14f53700cbf7a91f#internal